components/director/Dockerfile

```diff
@@ -1,8 +1,15 @@
-FROM golang:1.14.4-alpine3.12 as builder
+FROM golang:1.15.5-alpine3.12 as builder
 
 ENV BASE_APP_DIR /go/src/github.com/kyma-incubator/compass/components/director
 WORKDIR ${BASE_APP_DIR}
 
+#
+# Download dependencies
+#
+
+COPY go.mod go.sum ${BASE_APP_DIR}/
```

Comment:
line 31 in the dockerfile from alpine:edge -> FROM alpine:3.12.3 ( i guess there are some recent merged PRs in master taht are not applied here yet)